README.md

```diff
@@ -61,12 +61,16 @@ mas "1Password", id: 443987910
 whalebrew "whalebrew/wget"
 ```
 
-## Note
+## On Lockfiles
 
-Homebrew does not support installing specific versions of a library, only the most recent one, so there is no good mechanism for storing installed versions in a `.lock` file.
+Homebrew does not support installing specific versions of a library, only the most recent one, so there is no good mechanism for installing specific versions from a `.lock` file.
 
 If your software needs specific versions then perhaps you'll want to look at using [Vagrant](https://vagrantup.com/) to better match your development and production environments.
```

Comment:
While we're in here…
```suggestion
If your software needs specific versions, consider [Vagrant](https://vagrantup.com/), Docker (perhaps with [whalebrew](https://github.com/whalebrew/whalebrew), which homebrew-bundle supports), or another tool to install versions explicitly.
```

Reply:
```suggestion
Homebrew is a [rolling release](https://en.wikipedia.org/wiki/Rolling_release) package manager so does not support installing arbitrary older versions of software.
If your software needs specific pinned versions, consider [`whalebrew`](https://github.com/whalebrew/whalebrew) lines in your `Brewfile` to install [Docker](https://www.docker.com) containers).
```

Let's combine these a bit and remove mention of Vagrant which no-one seems to use any more.